packages/sanes-browser-extension/src/utils/test/navigation.ts

```diff
@@ -46,8 +47,8 @@ export const travelTo = async (
   return dom;
 };
 
-export const travelToWelcome = async (): Promise<React.Component> => {
-  return travelTo(WELCOME_ROUTE);
+export const travelToWelcome = async (hasPersona: boolean): Promise<React.Component> => {
+  return travelTo(WELCOME_ROUTE, [], null, hasPersona);
```

Comment:
you can set the 2 arguments to `undefined`, which is the same at not setting them. Then the caller does not need to bother what good default values are.